src/Onboarding/Onboarding.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * Onboarding class
+ *
+ * @package Give
+ */
+
+namespace Give\Onboarding;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Organizes WordPress actions and helper methods for Onboarding.
+ *
+ * @since 2.8.0
+ */
+class Onboarding {
+
+	/**
+	 * Initialize the Onboarding hooks/actions.
+	 *
+	 * @since 2.8.0
+	 */
+	public function init() {
+		add_action( 'admin_post_dismiss_setup_page', [ $this, 'dismissSetupPage' ] );
+	}
+
+	/**
+	 * Dissmiss the Setup Page.
+	 *
+	 * @since 2.8.0
+	 */
+	public function dismissSetupPage() {
+		if ( wp_verify_nonce( $_REQUEST['_wpnonce'], 'dismiss_setup_page' ) ) {
```

Comment:
@kjohnson Can we use specific Global param to access value. I think in this case it will be `$_GET`

Reply:
Happy to make the change. If I may, what is the issue with using $_REQUEST here? Specifying here would also require specifying on the form element - I'm not sure of the benefit.